lib/Models/GeoRssCatalogItem.js

```diff
@@ -0,0 +1,273 @@
+   * @memberOf GeoRssCatalogItem.prototype
+   * @type {DataSource}
+   */
+  dataSource: {
+    get: function() {
+      return defined(this._geoJsonItem)
+        ? this._geoJsonItem.dataSource
+        : undefined;
+    }
+  }
+});
+
+GeoRssCatalogItem.defaultUpdaters = clone(CatalogItem.defaultUpdaters);
+Object.freeze(GeoRssCatalogItem.defaultUpdaters);
+
+GeoRssCatalogItem.defaultSerializers = clone(CatalogItem.defaultSerializers);
+
+GeoRssCatalogItem.prototype._getValuesThatInfluenceLoad = function() {
+  return [this.url];
+};
+
+GeoRssCatalogItem.prototype._load = function() {
+  this._geoJsonItem = new GeoJsonCatalogItem(this.terria);
+  this._geoJsonItem.data = loadGeoRss(this);
+  this._geoJsonItem.name = this.name;
+  var that = this;
+
+  return this._geoJsonItem.load().then(function() {
+    if (!defined(that.rectangle)) {
+      that.rectangle = that._geoJsonItem.rectangle;
+    }
+  });
+};
+
+GeoRssCatalogItem.prototype._enable = function() {
+  if (defined(this._geoJsonItem)) {
+    this._geoJsonItem._enable();
+  }
+};
+
+GeoRssCatalogItem.prototype._disable = function() {
+  if (defined(this._geoJsonItem)) {
+    this._geoJsonItem._disable();
+  }
+};
+
+GeoRssCatalogItem.prototype._show = function() {
+  if (defined(this._geoJsonItem)) {
+    this._geoJsonItem._show();
+  }
+};
+
+GeoRssCatalogItem.prototype._hide = function() {
+  if (defined(this._geoJsonItem)) {
+    this._geoJsonItem._hide();
+  }
+};
+
+GeoRssCatalogItem.prototype.showOnSeparateMap = function(globeOrMap) {
+  if (defined(this._geoJsonItem)) {
+    return this._geoJsonItem.showOnSeparateMap(globeOrMap);
+  }
+};
+GeoRssCatalogItem.prototype.zoomTo = function() {
+  var that = this;
+  return when(this.load(), function() {
+    if (defined(that.nowViewingCatalogItem)) {
+      return that.nowViewingCatalogItem.zoomTo();
+    }
+
+    if (defined(that.rectangle)) {
+      return CatalogItem.prototype.zoomTo.call(that);
+    }
+
+    if (!defined(that.dataSource)) {
+      return;
+    }
+
+    return that.terria.currentViewer.zoomTo(that.dataSource);
+  });
+};
+
+function loadGeoRss(geoRssItem) {
+  return loadXML(proxyCatalogItemUrl(geoRssItem, geoRssItem.url)).then(xml => {
+    if (typeof xml === "string") {
+      const parser = new DOMParser();
+      xml = parser.parseFromString(xml, "text/xml");
+    }
+    var documentElement = xml.documentElement;
+    var promise;
+    if (documentElement.localName === "rss") {
+      promise = geoRss2Convertor(xml);
+      resolveCatalogMetaData(
+        documentElement.getElementsByTagName("channel")[0],
+        geoRssItem
+      );
+    } else if (documentElement.localName === "feed") {
+      promise = geoRssAtomConvertor(xml);
+      resolveCatalogMetaData(documentElement, geoRssItem);
+    } else {
+      throw new RuntimeError("document is not valid");
+    }
+    return promise;
+  });
+}
+
+function resolveCatalogMetaData(documentElement, geoRssItem) {
+  var children = documentElement.children;
```

Comment:
This isn't supported in IE11. I think it can be replaced with iteration over `childNodes`. See https://developer.mozilla.org/en-US/docs/Web/API/ParentNode/children (the "Polyfill" section shows how `childNodes` can be used similarly).

Reply:
Fixed